numba_dpex/tests/debugging/test_info.py

```diff
@@ -85,6 +88,34 @@ def test_info_args(
     for arg in expected_args:
         app.child.expect(arg)
 
+
+@pytest.mark.skip  # https://github.com/IntelPython/numba-dpex/issues/1223
```

Comment:
Is it leftover? I don't see any conditional breakpoints here.

Reply:
gdb.print() fails spuriously. https://github.com/IntelPython/numba-dpex/issues/1223